crates/cxx-qt-gen/src/parser/naming.rs

```diff
@@ -214,6 +214,20 @@ impl TypeNames {
             .cloned()
     }
 
+    /// Return a qualified version of the ident that can by used to refer to the type T outside of a CXX bridge
+    ///
+    /// Eg MyObject -> ffi::MyObject
+    ///
+    /// Note that this does not handle CXX types such as UniquePtr -> cxx::UniquePtr.
+    /// This is just for resolving impl T {} -> impl module::T {}
+    pub fn rust_qualified(&self, ident: &Ident) -> syn::Path {
```

Comment:
this was originally called `syn_ident_cxx_bridge_to_qualified_impl` and was specifically for the impl T type, so I think we should keep the `impl` name in the method otherwise it could easily be confused with `syn_type_cxx_bridge_to_qualified`

Reply:
As discussed in our meeting, this should be good to stay here.
I'll update the documentation on the function though.

Reply:
Done